source/features/allow-edits-unchecked-warning.tsx

```diff
@@ -0,0 +1,38 @@
+import {React} from 'dom-chef/react';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const generateWarning = () => (
+	<div class="flash flash-error" id="allow-edits-unchecked-warning">
+		<strong>Note:</strong> Disabling this would prevent the maintainer of this project from making vital changes to your branch.
+	</div>
+);
+
+function removeWarning() {
+	const warning = select('#allow-edits-unchecked-warning');
+	if (warning) {
+		warning.remove();
+	}
+}
+
+function init() {
+	const checkBox = select<HTMLInputElement>('.js-collab-option');
+
```

Comment:
What if the message is generated once here (no need for an extra function) and then it’s just appended and removed inside the event listener?

Reply:
That totally makes sense. Thanks for pointing that out.

Reply:
This is still outstanding

Reply:
Done. I originally thought you were talking about the `generateWarning` function. I've made the necessary fix.

Reply:
c892e92 is how I meant it